aim/tests/unit/agent/aid_universes/test_converter.py

```diff
@@ -213,6 +213,27 @@ class TestAciToAimConverterBD(TestAciToAimConverterBase, base.TestAimDBBase):
         l2_unknown_unicast_mode='flood')
 
 
+class TestAciToAimConverterNetflow(TestAciToAimConverterBase,
+                                   base.TestAimDBBase):
+    resource_type = resource.NetflowVMMExporterPol
+    reverse_map_output = [{'exceptions': {},
+                           'resource': 'netflowVmmExporterPol'}]
+    sample_input = [base.TestAimDBBase._get_example_aci_netflow(),
+                    base.TestAimDBBase._get_example_aci_netflow(
```

Comment:
We should use this one to overwrite those default parameters in the _get_example_aci_netflow().